webservices/partition/sched_a.py

```diff
@@ -40,6 +42,7 @@ def column_factory(cls, parent):
                 parent.c.contbr_id,
                 parent.c.cmte_id,
             ).label('clean_contbr_id'),
+            sa.cast(sa.func.get_transaction_year(parent.c[cls.transaction_date_column], parent.c.rpt_yr), sa.Numeric(4, 0)).label('transaction_two_year_period'),
```

Comment:
This was frustrating while I was working on this because I didn't think so either, but without it I end up seeing an error with the query that's ultimately executed saying that an unrecognized operation is taking place and that I may need to cast the value.  I isolated it to the entire call needing the wrap as opposed to individual values, but I agree, it shouldn't be needed in the first place given the return value.

I'll double check once I make the modifications to the SQL and try again to see if the same error occurs.